modules/s3_hosting/main.tf

```diff
@@ -14,12 +14,31 @@ locals {
     for domain in local.allDomains :
     domain if length(regexall("\\.", domain)) > 1
   ]
+
+  # allows frontend application to upload to pre-signed S3 urls
+  corsRules = var.cf_signed_downloads ? [{
```

Comment:
It's not clear from the variables that `allowed_cors_origins` will be ignored if `cf_signed_downloads` is false. We may want to allow people to supply this for any CF dist.